docs/source/usage.rst

```diff
@@ -311,17 +311,34 @@ Example:
 .. doctest::
```

Comment:
> I think we are missing documentation for interpolation of nodes:

I gave it a shot in 5f471ec
Feel free to add it elsewhere though (it's not directly related to this PR)

Reply:
> With the above, I can set the list in metrics_list without needing an interpolation syntax.

Makes sense. Are you suggesting to use that example instead of the one I currently have? (yours is a bit more complex since it involves a resolver, which comes later in the doc)

Reply:
> No, I am just thinking about the use case of list composition out-loud a bit.

Alright, resolving this conversation then.